NoiseEngine/Common/MemoryHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace NoiseEngine.Common;
+
+/// <summary>
+/// Collection of utility helpers memory management.
+/// </summary>
+internal static class MemoryHelper
```

Comment:
Use our [C# code style](https://github.com/NoiseStudio/docs/blob/master/csharp/CodeStyle.md). I mean branches, also use `System.Runtime.InteropServices` in using, don't use one line methods, and end file with aditional empty line.

Reply:
Fixed let me know if I missed something, Branch master was mistakenly assigned to this PR, there was one containing the naming format of other's that I've seen.